### PR TITLE
Add Habana 1.13 notebook

### PR DESCRIPTION
**Pull request by Xaenalt**
Habana currently added support for OpenShift 4.12.43 to their 1.13 driver series

Hopefully I did this update correctly

https://issues.redhat.com/browse/RHOAIENG-4343

### CODE REVIEW

**Comment by Xaenalt** [comment]
@harshad16 any idea what I'm doing wrong in the linting here?

**Comment by harshad16** [comment]
> any idea what I'm doing wrong in the linting here?

@Xaenalt I don't think you are doing anything wrong.
Its more to do with something from the past
i will take a look.

File: habana/1.13.0/ubi8-python-3.8/Pipfile [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+cffi = "==1.15.1"
+typing-extensions = ">=4.5.0"
+numpy = "==1.23.5"
+ninja = ">=1.11.1"
+pyyaml = "==6.0"
+pybind11 = "==2.10.4"
+mkl = "==2023.1"
+mkl-include = "==2023.1"
+
+# Parent image requirements to maintain cohesion
```
Comment (atheo89) [inline habana/1.13.0/ubi8-python-3.8/Pipfile:32]: The packages from line 33 to 58 have been updated to newer versions, could you take a look and update  this file as well? Ref link: https://github.com/opendatahub-io/notebooks/blob/main/jupyter/datascience/ubi8-python-3.8/Pipfile#L9-L45

**Review by atheo89 — CHANGES_REQUESTED** [review]
Thank you @Xaenalt for your pull request!
I added a comment for a small change that is needed.

**Pull request merged**